calico-enterprise_versioned_docs/version-3.17/network-policy/recommendations/policy-recommendations.mdx

```diff
@@ -150,24 +139,26 @@ To view policy-recommendation logs:
 kubectl logs -n tigera-policy-recommendation -l k8s-app=tigera-policy-recommendation
 ```
 
+**Notes**:
+This option is not available on managed clusters of a MCM instance.
```

Comment:
MCM is internal jargon and most CC users won't know about it. How about:

This is option is not available on managed clusters if you have implemented the multi-cluster-management feature.